app_config_b2c.py

```diff
@@ -1,14 +1,16 @@
 import os
 
 b2c_tenant = "fabrikamb2c"
-signupsignin_user_flow = "b2c_1_signupsignin1"
-editprofile_user_flow = "b2c_1_profileediting1"
-resetpassword_user_flow = "b2c_1_passwordreset1"
+signupsignin_user_flow = "B2C_1_signupsignin1"
```

Comment:
Then you would probably also want to update same strings in README_B2C.md

Reply:
Done! Ended up doing a full edit pass.